SSA/Projects/InstCombine/Base.lean

```diff
@@ -102,7 +118,22 @@ inductive MOp.BinaryOp : Type
   | sdiv
   | udiv
 deriving Repr, DecidableEq, Inhabited
-
+def aux : MOp.BinaryOp → ℕ → Lean.Format
```

Comment:
Similarly, `aux` is not descriptive at all: it tells me it's auxiliary to something, but I have no idea what the main thing is! This would better be names as `repr`, or maybe `reprAux` if you want to emphasize it's only a part of the `repr` implementation.